client/windows/CCastleInterface.h

```diff
@@ -42,6 +42,14 @@ class CBuildingRect : public CShowableAnim
 {
 	std::string getSubtitle();
 public:
+	enum EBuildingCreationAnimationPhases : uint32_t

```

Comment:
minor suggestion - make it private

Reply:
it wont work because another class uses it